packages/location-field/src/index.tsx

```diff
@@ -954,7 +954,7 @@ const LocationField = ({
       </S.HiddenContent>
       <ItemList
         // Hide the list from screen readers if no enabled options are shown.
-        aria-hidden={hasNoEnabledOptions}
+        aria-hidden={hasNoEnabledOptions || !menuVisible}
```

Comment:
I noted the `fix`commit for this change, which will trigger a release for this package.